### PR TITLE
Explicit unicode support and command-line arguments.

### PR DESCRIPTION
**Pull request by MrAksel**
Changed FindWindowA to FindWindowW, and changed to use explicit unicode text literals in the code. Implemented command line options to modify the appearance of the taskbar. These are "--blur", "--opaque", "--transparent" and "--tint COLOR", and are documented in the code.

### CODE REVIEW

File: TranslucentTB/main.cpp [hunk 1]
```diff
@@ -16,32 +17,170 @@ struct WINCOMPATTRDATA
 	ULONG ulDataSize;
 };
 
+struct OPTIONS
+{
+	int mode;
```
Comment (ethanhs) [inline TranslucentTB/main.cpp]: Could you change this from `mode` to something a bit more descriptive such as `blur` or something? Thanks!

**Comment by MrAksel** [comment]
This should solve those problems! I wrote an explanation for the --tint parameter, and improved the PrintHelp() function. It is somewhat cluttered and overly complicated, so I'll just move it out into usage.md later and let you edit it to your wishes (see the bug I've commented in the code). Also renamed the field you talked about and added some descriptive constants for this field. Should read a little better now.

**Comment by charlesmilette** [comment]
Perhaps I could look into implementing a settings file since this pull request basically adds the architecture for that. Instead of reading from the command line, I'd simply read from a file. Also an argument to ignore the config file would be useful when testing.

**Comment by MrAksel** [comment]
I was thinking about the same, as it would be easier to edit for the common user than applying arguments to the program on startup. I'm already stretching my limits on C++ though, as it isn't exactly my native language, so I'll leave the config file up to you. 

On another matter, now that I added usage.md, should I just remove the PrintHelp() function? It doesn't help much, and if you want to step away from command-line arguments its not much point in having it there anyway.

**Comment by ethanhs** [comment]
Command line arguments is something I want to support, at least for now. I also want a config file. I think the command line allows greater exploration than a settings file, so I think the program should have both. Thanks so much @MrAksel this is awesome.

**Pull request merged**